### PR TITLE
feat: add text for food and childcare

### PR DESCRIPTION
**Pull request by 2Judy2**
Feel free to comment for changes :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -168,6 +169,39 @@ <h3>## Hamburg</h3>
...
+        <img src="assets/sticker/12.png" />
+      </div>
+
+      <h2># Offers</h2>
+
+      <div>
+        <h3>## Food</h3>
+
+        <p>
+          This year we only want to provide vegetarian food like last year. In the morning there will be breakfast with sandwiches.
```
Comment (alster-wasser) [inline index.html]: (sorry for the late reaction)
the food is actually vegan, not just vegetarian!

File: index.html [hunk 2]
```diff
@@ -168,6 +169,39 @@ <h3>## Hamburg</h3>
...
+          After the first talks there will be a vegetarian lunch and in the afternoon you can enjoy (oder vielleicht sweeten) your
```
Comment (alster-wasser) [inline index.html]: I'd go with enjoy

File: index.html (hunk 1 above)
Comment (2Judy2) [reply index.html]: Done :)

File: index.html (hunk 2 above)
Comment (2Judy2) [reply index.html]: Done :)

**Comment by alster-wasser** [comment]
LGTM

**Review by alster-wasser — APPROVED** [review]

**Pull request merged**